.travis.yml

```diff
@@ -2,18 +2,17 @@ sudo: false
 language: php
 
 php:
-    - 7.0
-    - 7.1
-    - 7.2
     - 7.3
+    - 7.4
+    - nightly
     - hhvm
 
 matrix:
     allow_failures:
         - php: hhvm
 
 before_script:
-    - composer self-update
-    - composer install --prefer-dist --no-interaction
+    - composer self-update --2
+    - composer install --prefer-dist --no-interaction --ignore-platform-req=php
```

Comment:
`--ignore-platform-req` is required here to get PHPUnit to install on PHP 8

Reply:
> Also, why is this required exactly? Is it because PHPUnit does not support PHP 8 yet?

It's due to some dependencies, E.G PHPUnit requires `phpspec/prophecy` which doesn't have a version that supports PHP 8 yet